JavascriptMornings/Maitee
# Add package json

## Review checklist

- [x] Add corresponding label

## Testing

Steps to test the change:
- Run `npm install or npm i` in the project directory and make sure we have the `node_modules` folder created with Jest installed

Remember that in order to use npm(node package manager) you need to install `nodejs`

## Additional notes
This PR is in order to add a `package.json` file to our project since probably in the future we will need to use functions from an external library, i'm adding `Jest` to encourage you guys of making test of any new functionality added to the project this is a good practice that we all as developer should be following.

Also i'm adding jest because i noticed we have a patter to test all the functionality in the `index.js` and it's better to add those test examples using `jest`.

If you have any questions/recommendations just let me know. Will make a follow PR adding the test and removing all the changes we have in the `index.js` file

Information about `package.json` can be found here https://docs.npmjs.com/files/package.json
Information related to `Jest`  https://facebook.github.io/jest/
`.gitignore` https://git-scm.com/docs/gitignore
## Deploy checklist
